sentry_sdk/integrations/logging.py

```diff
@@ -222,7 +222,20 @@ def _emit(self, record):
 
         event["level"] = _logging_to_event_level(record.levelname)
         event["logger"] = record.name
-        event["logentry"] = {"message": to_string(record.msg), "params": record.args}
+
+        # Ensure warnings captured by logging module arrive as separate issues
+        # Before https://bugs.python.org/issue46557 was addressed, captured
+        # warnings were logged with logger.warning("%s", s) which caused all
+        # captured warnings to be grouped under the same issue
+        # Refer to https://docs.python.org/3/library/logging.html#logging.captureWarnings
```

Comment:
A little bit to specific. In a couple of months/years no one cares about the python issue. Please change to something like "Capture warnings from `warnings` module when logging.captureWarnings(True) was set."

Reply:
Done ✅

Reply:
ok